operations/configuration_management/CFS_Global_Options.md

```diff
@@ -5,101 +5,151 @@ The Configuration Framework Service \(CFS\) provides a global service options en
   This back-off time will continue to increase up to this cap, and will reset to 0 when a new session is successful.
 
-* **`batcherPendingTimeout`**
+  See [Automatic Configuration Management](Automatic_Configuration_Management.md) for more information.
+
+* **`batcher_pending_timeout`**
 
   This option specifies the maximum number of seconds that CFS batcher will wait for a new session to enter a running state before deleting the session and trying again.
   This retry helps manage rare communication errors that can cause sessions to be stuck in a pending state.
 
-* **`defaultAnsibleConfig`**
+* **`debug_wait_time`**
+
+  Any sessions that have failed while using the `debug_on_failure` option will wait for this many seconds before automatically terminating if the the completion flag
+  is not touched.
 
-  See [Set the `ansible.cfg` for a Session](Set_the_ansible-cfg_for_a_Session.md) for more information.
+* **`default_ansible_config`**
 
-* **`defaultBatcherRetryPolicy`**
+  See [Configure Ansible](Configure_Ansible.md) for more information.
 
-  When a component \(node\) requiring configuration fails to configure from a previous configuration session launched by CFS batcher, the error is logged.
-  `defaultBatcherRetryPolicy` is the maximum number of failed configurations allowed per component before CFS batcher will stop attempting to configure the component.
+* **`default_batcher_retry_policy`**
 
-  See [Configuration Management with the CFS Batcher](Configuration_Management_with_the_CFS_Batcher.md) for more information.
+  When a component requiring configuration fails to configure, the error is logged.
+  `default_batcher_retry_policy` is the maximum number of failed configurations allowed per component before CFS batcher will stop attempting to configure the component.
+  This value can be overridden on a per component basis.
 
-* **`defaultPlaybook`**
+  See [Automatic Configuration Management](Automatic_Configuration_Management.md) for more information.
+
+* **`default_page_size`**
+
+  > **`NOTE`** This option is only available in the v3 CFS API.
+
+  When list listing component, session or configuration records, CFS will only return a limited number of records.
+  If the `limit` parameter is not specified in the call, the `default_page_size` is what will be used for the maximum number of records returned in one call.
+  See [Paging CFS Records](Paging_CFS_Records.md) for more information on paging.
+
+  This parameter has a secondary purpose in the v2 API.
+  Calls that exceed this number of records will instead return an error stating that the response is too large.
+  In this case users should switch to the v3 API to take advantage of paging.
+  This number can be increased through the v3 API to allow more records to be returned through the v2 API, but this risks causing calls to fail due to memory constraints.
+
+* **`default_playbook`**
+
+  > **WARNING:** This option is deprecated in the v3 CFS API.
```

Comment:
Do you think it makes sense to elaborate here?

```suggestion
  > **WARNING:** This option is deprecated and read-only in the v3 CFS API. It can only be modified using CFS v2.
```